packs/kingmaker-bestiary/phantasmal-protagonist.json

```diff
@@ -0,0 +1,268 @@
+{
+    "_id": "5yd6BBLY2vCNz1zh",
+    "img": "systems/pf2e/icons/default-icons/npc.svg",
+    "items": [
+        {
+            "_id": "R7dn3GjbAQ64z6a4",
+            "img": "systems/pf2e/icons/default-icons/spellcastingEntry.svg",
+            "name": "Divine Innate Spells",
```

Comment:
We don't need the spellcasting entry. The DC of the inline save is based on the caster anyways, so we should just cut that inline